lib/redis/store/namespace.rb

```diff
@@ -43,6 +43,10 @@ def del(*keys)
         super(*keys.map {|key| interpolate(key) }) if keys.any?
       end
 
+      def watch *keys
```

Comment:
can you put parens around this method argument? just to match the rest of the file.

Reply:
Done